elastic/integrations
# Adding HTTP metrics collection to Kibana integration

## What does this PR do?


Updates the Kibana integration to use the metricbeat HTTP module to collect background task utilization metrics from the Kibana API endpoint added in [this PR](https://github.com/elastic/kibana/pull/153600).

## Checklist


## Author's Checklist



## How to test this PR locally

- Run the stack using `elastic-package`: `elastic-package stack up --version=8.9.0-SNAPSHOT -v -d`
- Build the package:
  - `cd integrations/packages/kibana`
  - `elastic-package build`
- Deploy the package: `elastic-package stack up -v -d --services package-registry`
- Navigate to https://localhost:5601/app/integrations/browse
- Select and add the Kibana integration
- Verify you see the new `HTTP metrics background task utilization` integration
- Configure the integration for host `https://kibana:5601` and username/password `elastic:changeme`
- Verify you see a datastream for `metrics-kibana.background_task_utilization-default`
- Verify you see data in the datastream (`GET metrics-kibana.background_task_utilization-default/_search`) with the `kibana.background_task_utilization.stats.value.load` field populated



## Related issues


- https://github.com/elastic/kibana/issues/155779

## Screenshots



<img width="861" alt="Screenshot 2023-05-11 at 1 44 12 PM" src="https://github.com/elastic/integrations/assets/13104637/a3aba8f4-1e59-453d-859d-3d05bf1333fb">
